src/dune_pkg/opamUrl0.mli

```diff
@@ -4,7 +4,7 @@ val equal : t -> t -> bool
 val hash : t -> int
 val to_string : t -> string
 val of_string : string -> t
-val decode : t Dune_sexp.Decoder.t
+val decode : (Stdune.Loc.t * t) Dune_sexp.Decoder.t
```

Comment:
Is it common that `decode` would return a tuple of `loc * t`?

Reply:
I suppose we call it `decode_loc` elsewhere. I renamed it